developer/src/kmc-package/test/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.kmp.intermediate.json

```diff
@@ -16,7 +16,7 @@
       "description": "SENĆOŦEN (Saanich Dialect) Lexical Model"
     },
     "version": {
-      "description": "1.0.3"
+      "description": "0.0.0"
     }
```

Comment:
I am not sure this is right. `options.followKeyboardVersion` is not set, so the version number is relevant.

I think we can chase this as a new issue in the package compiler given I am doing work on it anyway, and it probably relates to this work.

(I'll just say 'ditto' for other unit tests)

Reply:
+1